### PR TITLE
implement tenant-config externalization

### PR DESCRIPTION
**Pull request by sergeysenja1992**

### CODE REVIEW

File: src/test/resources/tenant-config.yml [hunk 1]
```diff
@@ -0,0 +1,222 @@
+inviteExpireTime: 86400
```
Comment (ishkurko) [inline src/test/resources/tenant-config.yml]: please, user general file

Comment (sergeysenja1992) [reply src/test/resources/tenant-config.yml]: поправил

File: src/test/resources/logback.xml [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <configuration scan="true">
     <include resource="org/springframework/boot/logging/logback/base.xml"/>
 
-    <logger name="com.icthh.xm.ms.configuration" level="INFO"/>
+    <logger name="com.icthh.xm.ms.configuration" level="TRACE"/>
```
Comment (ishkurko) [inline src/test/resources/logback.xml]: Do we need the trace for a test?

File: src/test/java/com/icthh/xm/ms/configuration/service/processors/TenantConfigExternalizationTest.java [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+        return map;
+    }
+
+    @SneakyThrows
+    public static String loadFile(String path) {
+        InputStream cfgInputStream = new ClassPathResource(path).getInputStream();
+        return IOUtils.toString(cfgInputStream, UTF_8);
+    }
+
+}
```
Comment (ishkurko) [inline src/test/java/com/icthh/xm/ms/configuration/service/processors/TenantConfigExternalizationTest.java]: Please, add new line.

**Pull request merged**